.github/workflows/release.yaml

```diff
@@ -45,3 +45,43 @@ jobs:
           asset_path: manifests/okd/kubevirt-tekton-tasks-okd.yaml
           asset_name: kubevirt-tekton-tasks-okd.yaml
           asset_content_type: text/plain
+      - name: Update tekton tasks manifests
+        run: |
+          # Define vars
+          export VERSION=${{ steps.get_release.outputs.tag_name }}
+          # If GITHUB_FORK_USER is changed, a new access token should be set as a repo secret (ACTIONS_TOKEN)
```

Comment:
is this necessary? Can't we just create a new branch in the main repo to use for the PR. Or just push to the main branch?

Reply:
or alternatively we could just keep it simple and make a PR with regenerated manifests manually before the release?

I suppose we could do an opposite approach. Manually create a release after change in version is pushed to the main branch.